jsk_robot_common/jsk_robot_startup/launch/biped_localization.launch

```diff
@@ -8,52 +8,70 @@
   <arg name="slam_laser_max_height" default="0.2"/>
   <arg name="slam_laser_min_height" default="-0.2"/>
   <arg name="robot" default="default"/>
-
+  
   <!-- Add slam laser pointcloud nodes to multisense_laser nodelet -->
-  <group if="$(arg set_slam_laser_params)">
-    <include file="$(find jsk_robot_startup)/launch/slam_laser_nodelets.launch">
-      <arg name="nodelet_index" value="$(arg nodelet_index)"/>
-      <arg name="laser_input" value="/multisense/cloud_self_filtered"/>
-      <arg name="assemble_base_frame" value="/biped_odom_particle" if="$(arg use_particle_odom)"/>
-      <arg name="assemble_base_frame" value="/odom_init" unless="$(arg use_particle_odom)"/>      
-    </include>
-  </group>
-
-  <!-- viso -->
-  <include file="$(find jsk_robot_startup)/launch/viso.launch">
-    <arg name="stereo" default="$(arg stereo_namespace)" />
-    <arg name="image" default="image_rect" />
-    <arg name="use_robot_pose_ekf" default="false" />
-    <arg name="publish_viso_tf" default="false" />
-    <arg name="invert_viso_tf" default="true" /> 
+  <include file="$(find jsk_robot_startup)/launch/slam_laser_nodelets.launch" if="$(arg set_slam_laser_params)">
+    <arg name="nodelet_index" value="$(arg nodelet_index)"/>
+    <arg name="laser_input" value="/multisense/cloud_self_filtered"/>
+    <arg name="assemble_base_frame" value="/biped_odom_particle" if="$(arg use_particle_odom)"/>
```

Comment:
I think frame starting with / is not allowed with tf2.
in 14.04 environment, we have the error like this
```
Warning: Invalid argument "/biped_odom_particle" passed to canTransform argument source_frame in tf2 frame_ids cannot start with a '/' like: 
         at line 130 in /tmp/buildd/ros-indigo-tf2-0.5.12-0trusty-20151111-0404/src/buffer_core.cpp
[ERROR] [1453876106.893551384, 36.746000000]: Transform failure: Invalid argument "/biped_odom_particle" passed to lookupTransform argument source_frame in tf2 frame_ids cannot start with a '/' like: 
```
refer to these lines https://github.com/ros/geometry_experimental/blob/indigo-devel/tf2/src/buffer_core.cpp#L146